install_kafka.yml

```diff
@@ -0,0 +1,92 @@
+---
+  - name: "Install Kafka"
+    hosts: kafkaserver1
+
+
+    roles:
+    - install-JDK-role
+
+    tasks:
+
+      #  - name: Install the latest version of JDK
+      #    become: yes
+      #    yum:
+      #     name: jdk-11
+      #     state: latest
+
+    
+       - name: Copy Kafka Binaries
+         become: yes 
+         become_user: dojokafka
+         copy:
+           src: /Users/rafiahmadjan/Documents/DOJO/Dojo-Mini-Kafka/files/kafka_2.13-3.2.0.tgz
+           dest: '{{ home }}'
+           owner: dojokafka
+           group: dojo
+           mode: 0644
+         
+
+       - name:  untar the binaries
+         become: yes 
+         become_user: dojokafka
+         ansible.builtin.unarchive:
+           src: "{{ home }}/kafka_2.13-3.2.0.tgz"
+           dest: '{{ home }}'
+           remote_src: yes
+
+
+       - name: Create a symbolic link
+         become: yes 
+         become_user: dojokafka
+         ansible.builtin.file:
+           src: kafka_2.13-3.2.0
+           dest: '{{ base }}'
+           owner: dojokafka
+           group: dojo
+           state: link
+           
+      
+            
+
+      #  - name: create link
+        #  become: yes 
+        #  become_user: dojokafka
+        #  shell: ln -s kafka_2.13-3.2.0 kafka 
+        #  args:
+        #    chdir: /home/dojokafka/
+        #    warn: true
+
+#       - name: Start Kafka
+#         shell: bin/zookeeper-server-start.sh config/zookeeper.properties > /home/dojokafka/kafka-start.log  2>&1 &
+#         args:
+#          chdir: /home/dojokafka/kafka
+#          warn: false
+      
+       - name: Copy service files
+         become: yes 
+         copy: 
```

Comment:
`copy` -> `template`   
see prev comment

Reply:
Used templates instead of copying files